js/utility/binarySearch.js

```diff
@@ -0,0 +1,31 @@
+namespacer('baltimoreCounty.utility');
+
+baltimoreCounty.utility.binarySearch = (function binarySearchClosure() {
+	'use strict';
+
+	var binarySearch = function binarySearch(list, item) {
```

Comment:
probably don't need the variable declaration here

Reply:
Technically I don't, but it's consistent with everywhere else we use it.